internal/endorser/endorser.go

```diff
@@ -58,14 +57,14 @@ func loadAndVerifyProvenances(provenanceURIs []string, referenceValues verifier.
 	}
 
 	// load provenances from URIs
-	provenances := make([]slsa.ValidatedProvenance, 0, len(provenanceURIs))
+	provenances := make([]amber.ValidatedProvenance, 0, len(provenanceURIs))
 	provenancesData := make([]amber.ProvenanceData, 0, len(provenanceURIs))
 	for _, uri := range provenanceURIs {
 		provenanceBytes, err := getProvenanceBytes(uri)
 		if err != nil {
 			return nil, fmt.Errorf("couldn't load the provenance file from %s: %v", uri, err)
 		}
-		provenance, err := slsa.ParseProvenanceData(provenanceBytes)
+		provenance, err := amber.ParseProvenanceData(provenanceBytes)
```

Comment:
I think it would be nicer to also replace this with a generic function that returns a `ProvenanceIR`. So we don't have an explicit dependency to `slsa`, `amber` or any other package. Would that work?

Reply:
I think that would be a good idea. I would propose this in a different PR.